addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -90,8 +90,29 @@ public class AuthUtils {
     public static final String[] HEADERS = {HttpHeader.AUTHORIZATION};
     public static final String[] JSON_IDS = {"accesstoken", "token"};
     private static final String[] USERNAME_FIELD_INDICATORS = {
-        "email", "signinname", "uname", "user"
+        "email", "signinname", "uname", "user", "name", "nome", "nombre"
```

Comment:
I believe we've seen username and uid elsewhere

Reply:
Before adding more we should check if they are really needed.

Reply:
Yup that's fair.